frontend/public/kubevirt/components/nic.jsx

```diff
@@ -19,7 +19,7 @@ const getNetworkType = network => {
 const menuActionDelete = (vm, nic) => ({
   label: 'Delete',
   callback: () => deleteDeviceModal({
-    type: NETWORK,
+    type: NIC,
     device: nic,
     vm: vm
   })
```

Comment:
The NicHeader should IMO rather be 
Name | Model | Network | MAC Address

Reply:
I forgot the headers are automatically converted to upper case, but I would still change it to MAC for the sake of the code.